integration/sawtooth_integration/tests/test_identity.py

```diff
@@ -0,0 +1,399 @@
+    return "{},{},{}".format(unique_value, 'create', '').encode('utf-8')
+
+
+def XO_ENCODER(contents):
+    return contents
+
+
+def XO_ADDRESS(unique_value):
+    return XO_NAMESPACE + MessageFactory.sha512(unique_value.encode())[:64]
+
+
+TransactionEncoder = {
+    Families.INTKEY: {
+        'encoder': cbor.dumps,
+        'payload_func': INTKEY_PAYLOAD,
+        'address_func': INTKEY_ADDRESS
+    },
+    Families.XO: {
+        'encoder': XO_ENCODER,
+        'payload_func': XO_PAYLOAD,
+        'address_func': XO_ADDRESS
+    }
+}
+
+
+class Transactor(object):
+
+    def __init__(self, name, rest_endpoint):
+        """
+        Args:
+            name (str): An identifier for this Transactor
+            rest_endpoint (str): The rest api that this Transactor will
+                communicate with.
+        """
+
+        self.name = name
+        self._rest_endpoint = rest_endpoint \
+            if rest_endpoint.startswith("http://") \
+            else "http://{}".format(rest_endpoint)
+        with open('/root/.sawtooth/keys/{}.priv'.format(name)) as priv_file:
+            private_key = priv_file.read().strip('\n')
+        self._private_key = private_key
+        with open('/root/.sawtooth/keys/{}.pub'.format(name)) as pub_file:
+            public_key = pub_file.read().strip('\n')
+        self._public_key = public_key
+        self._factories = {}
+        self._client = RestClient(url=self._rest_endpoint)
+
+        self._add_transaction_family_factory(Families.INTKEY)
+        self._add_transaction_family_factory(Families.XO)
+
+    @property
+    def public_key(self):
+        return self._public_key
+
+    def _add_transaction_family_factory(self, family_name):
+        """Add a MessageFactory for the specified family.
+
+        Args:
+            family_name (Families): One of the Enum values representing
+                transaction families.
+        """
+
+        family_config = FamilyConfig[family_name]
+        self._factories[family_name] = MessageFactory(
+            family_name=family_config['family_name'],
+            family_version=family_config['family_version'],
+            encoding=family_config['encoding'],
+            namespace=family_config['namespace'],
+            private=self._private_key,
+            public=self._public_key)
+
+    def create_txn(self, family_name, batcher=None):
+        unique_value = uuid4().hex[:20]
+        encoder = TransactionEncoder[family_name]['encoder']
+        payload = encoder(
+            TransactionEncoder[family_name]['payload_func'](unique_value))
+
+        address = TransactionEncoder[family_name]['address_func'](unique_value)
+
+        return self._factories[family_name].create_transaction(
+            payload=payload,
+            inputs=[address],
+            outputs=[address],
+            deps=[],
+            batcher=batcher)
+
+    def create_batch(self, family_name):
+        transactions = [self.create_txn(family_name) for _ in range(1)]
```

Comment:
`range(1)` should be `range(count)` for an argument `count`, or else just make this `[self.create_txn(family_name]`.

Reply:
I will go with the former, with a default variable.